docs/image-format.md

```diff
@@ -23,6 +23,9 @@ import CardTextResult from '@site/image-generator/yml/generator-docs/card-text.y
 import TrashSource from '!raw-loader!@site/image-generator/yml/generator-docs/trash.yml';
 import TrashResult from '@site/image-generator/yml/generator-docs/trash.yml';
 
```

Comment:
no space between imports

Reply:
I disagree. Look above.
These imports are paired source/result

Reply:
ok, i take it back